cardano-cli/src/Cardano/CLI/Types/Errors/TxValidationError.hs

```diff
@@ -338,36 +337,15 @@ convertToTxVotingProcedures
  :: [(VotingProcedures era, Maybe (ScriptWitness WitCtxStake era))]
  -> Either (TxGovDuplicateVotes era) (TxVotingProcedures BuildTx era)
 convertToTxVotingProcedures votingProcedures = do
-  VotingProcedures procedure <- foldM f emptyVotingProcedures votingProcedures
+  VotingProcedures procedure <- first TxGovDuplicateVotes $
+    foldM f emptyVotingProcedures votingProcedures
```

Comment:
You should be using `mergeVotingProcedures` from `cardano-api`.

Reply:
`f` is using it. What should be changed here?